.github/workflows/apk.yml

```diff
@@ -29,12 +29,15 @@ jobs:
           distribution: 'zulu'
           java-version: '17'
 
-      # Load google-services.json and local.properties from the secrets
-      - name: Decode google-services.json
+      # Load google-services.json
+      - name: Decode google-services.json and google maps api key and add to system env
         env:
+          GOOGLE_MAPS_API_KEY: ${{secrets.GOOGLE_MAPS_API_KEY}}
           GOOGLE_SERVICES: ${{ secrets.GOOGLE_SERVICES }}
         run: |
           echo "$GOOGLE_SERVICES" | base64 --decode > ./app/google-services.json
+          MAPS_API_KEY=$(echo "$GOOGLE_MAPS_API_KEY" | base64 --decode)
+          echo "MAPS_API_KEY=$MAPS_API_KEY" >> $GITHUB_ENV
```

Comment:
Hi, the key isn't actually being printed onto the screen! No worries, it's being piped into base64. The echo is there so that the key becomes a stream of characters into the base64 decoding.